datasets/at/meine_abgeordneten/crawler.py

```diff
@@ -200,7 +200,7 @@ def crawl(context: Context):
     response = context.fetch_html(context.data_url)
 
     # XPath to the url for the pages of each politician
-    xpath_politician_page = '//*[contains(@class, "abgeordneter")]/*/a/@href'
+    xpath_politician_page = '//div[(@class="abgeordneter row")]/*/a/@href'
```

Comment:
why are we making this more strict? I think checking for row as well is fine, but we shouldn't make assumptions about what else is in the class attribute, or what the ordering of classes is. So rather 

```
//div[contains(@class, "abgeordneter")][contains(@class, "row")]/*/a/@href